pkg/client/init.go

```diff
@@ -233,6 +239,29 @@ func InClusterClientForKarmadaApiServer() kubeclient.Interface {
 	return inClusterClientForKarmadaApiServer
 }
 
+func InClusterClientForKarmadactlApiServer(clustername string) kubeclient.Interface {
+	if !isKarmadaInitialized() {
+		return nil
+	}
+	karmadaCtlRestConfig, _, err := GetKarmadaConfig()
+	if strings.Contains(karmadaCtlRestConfig.Host, clustername) {
+		return inClusterClientForKarmadactlApiServer
+	} else {
+		karmadaCtlRestConfig.Host = clusterHost + "/apis/cluster.karmada.io/v1alpha1/clusters/" + clustername + "/proxy"
+	}
+	if err != nil {
+		klog.ErrorS(err, "Could not get karmadactl restConfig")
+		return nil
+	}
+	c, err := kubeclient.NewForConfig(karmadaCtlRestConfig)
+	if err != nil {
+		klog.ErrorS(err, "Could not init kubernetes in-cluster client for karmadactl apiserver")
+		return nil
+	}
+	inClusterClientForKarmadactlApiServer = c
```

Comment:
client for member cluster need multi instances, maybe use map to store the client ?